docs/streams/stream-basics/subscriptions.md

```diff
@@ -1,21 +1,40 @@
 ---
-sidebar_position: 40
+sidebar_position: 200
 title: Subscriptions
 ---
 
 A subscription is a named configuration rule that determines how messages are delivered to consumers. There are three available subscription modes in GDN streams: [exclusive](#exclusive), [shared](#shared), and [failover](#failover). These modes are illustrated in the figure below.
```

Comment:
Add key_shared link to this list. Also, talk to @matt-macrometa about improving these images.